client/common/directives/ModelField/ModelField.js

```diff
@@ -263,12 +263,13 @@ angular.module('dashboard.directives.ModelField', [
           </div>';
         break;
       case 'number':
-        template = '<label class="col-sm-2 control-label">{{ display.label || key }}:</label>\
-          <div class="col-sm-10">\
-            <div class="error-message" ng-if="display.error.length > 0">{{ display.error }}</div>\
-            <model-field-number key="key" property="property" options="display.options" ng-required="model.properties[key].required" ng-disabled="display.readonly" model-data="data" ng-model="data[key]" ng-error="onFieldError(error)" class="field" ng-class="{error: display.error.length > 0}" />\
-            <div class="model-field-description" ng-if="display.description">{{ display.description }} {{count}}</div>\
-          </div>';
+        template = '<model-field-number key="key" property="property" options="display.options" ng-required="model.properties[key].required" ng-disabled="display.readonly" model-data="data" ng-model="data[key]" ng-error="onFieldError(error)" class="field" ng-class="{error: display.error.length > 0}" />';
```

Comment:
Reverted, but I had to move the ng-class for error styling into the directive in order to get descriptions to display. Updated styl file appropriately.